exercises/concept/rpn-calculator-inspection/.docs/instructions.md

```diff
@@ -0,0 +1,85 @@
+# Instructions
+
+Your work at _Instruments of Texas_ on an experimental RPN calculator continues. Your team has built a few prototypes that need to undergo a thorough inspection, to choose the best one that can be mass-produced.
+
+You want to conduct two types of checks.
+
+Firstly, a reliability check that will detect inputs for which the calculator under inspection either crashes or doesn't respond fast enough. To isolate failures, the calculations for each input need to be run in a separate process. Linking and trapping exits in the caller process can be used to detect if the calculation finished or crashed.
+
+Secondly, a correctness check that will check if for a given input, the result returned by the calculator is as expected. Only calculators that already passed the reliability check will undergo a correctness check, so crashes are not a concern. However, the operations should be run concurrently to speed up the process, which makes it the perfect use case for asynchronous tasks.
+
+## 1. Start a reliability check for a single input
+
+Implement the `RPNCalculatorInspection.start_reliability_check/2` function. It should take 2 arguments, a function (the calculator), and an input for the calculator. It should return a map that contains the input and the PID of the spawned process.
+
+The spawned process should call the given calculator function with the given input. The process should be linked to the caller process.
+
+```elixir
+RPNCalculatorInspection.start_reliability_check(fn _ -> 0 end, "2 3 +")
+# => %{input: "2 3 +", pid: #PID<0.169.0>}
+```
+
+## 2. Interpret the results of a reliability check
+
+Implement the `RPNCalculatorInspection.await_reliability_check_result/2` function. It should take two arguments. The first argument is a map with the input of the reliability check and the PID of the process running the reliability check for this input, as returned by `RPNCalculatorInspection.start_reliability_check/2`. The second argument is a map that serves an accumulator for the results of reliability checks with different inputs.
+
+The function should wait for an exit message.
+
+If it receives an exit message (`{:EXIT, from, reason}`) with the reason `:normal` from the same process that runs the reliability check, it should return the results map with the value `:ok` added under the key `input`.
+
+If it receives an exit message with a different reason from the same process that runs the reliability check, it should return the results map with the value `:error` added under the key `input`.
+
+If it doesn't receive any messages in 100ms, it should return the results map with the value `:timeout` added under the key `input`.
+
+```elixir
+send(self(), {:EXIT, self(), :normal})
```

Comment:
This might be misleading to have this placed here
```suggestion
```

Reply:
I put it there to show that there is a message waiting in the process inbox. Without, there won't be a difference between the two different function call examples. I'm not sure how to show this better?

Reply:
maybe then a comment to annotate the purpose of it being there?

Reply:
Maybe like this [`58a4292` (#682)](https://github.com/exercism/elixir/pull/682/commits/58a4292b16679a18f08aeea9ebd0eda672f157a6) ?

Reply:
👍🏻